File: app/src/main/java/buggy/plandroid/com/plandroidbugs/UserWire.java
package buggy.plandroid.com.plandroidbugs;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Map;

@JsonIgnoreProperties(ignoreUnknown = true)
public class UserWire {

    @JsonProperty(value = "uid") public String userId;
    @JsonProperty(value = "email") public String email;
    @JsonProperty(value = "first_name") public String firstName;
    @JsonProperty(value = "last_name") public String lastName;
    @JsonProperty(value = "company") public String company;
    @JsonProperty(value = "title") private String title;
    @JsonProperty(value = "language") public String language;
    @JsonProperty(value = "role") public Role role;
}

File: app/src/main/java/buggy/plandroid/com/plandroidbugs/UserWire.kt
package buggy.plandroid.com.plandroidbugs

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class UserWire(
    @JsonProperty(value = "uid")
    val userId: String,

    @JsonProperty(value = "email")
    val email: String,

    @JsonProperty(value = "first_name")
    val firstName: String,

    @JsonProperty(value = "last_name")
    val lastName: String,

    @JsonProperty(value = "company")
    val company: String,

    @JsonProperty(value = "title")
    val title: String,

    @JsonProperty(value = "language")
    val language: String,
)